File: backend/src/main/java/com/project/dto/professional/response/ProfessionalAvailabilityResponseDTO.java
package com.project.dto.professional.response;


import java.time.LocalTime;

import com.project.enums.DayOfWeekEnum;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class ProfessionalAvailabilityResponseDTO {

    private Long availabilityId;
    private DayOfWeekEnum dayOfWeek;
    private LocalTime startTime;
    private LocalTime endTime;
}

File: backend_dotnet/SafeMind.Core/DTOs/Professional/Response/ProfessionalAvailabilityResponseDTO.cs
using SafeMind.Core.Enums;

namespace SafeMind.Core.DTOs.Professional.Response
{
    public class ProfessionalAvailabilityResponseDTO
    {
        public long AvailabilityId { get; set; }
        public DayOfWeekEnum DayOfWeek { get; set; }
        public string StartTime { get; set; } = string.Empty;
        public string EndTime { get; set; } = string.Empty;

        public ProfessionalAvailabilityResponseDTO(long availabilityId, DayOfWeekEnum dayOfWeek, TimeSpan startTime, TimeSpan endTime)
        {
            AvailabilityId = availabilityId;
            DayOfWeek = dayOfWeek;
            StartTime = startTime.ToString(@"hh\:mm");
            EndTime = endTime.ToString(@"hh\:mm");
        }
    }
}
